wg-charter/01-01

```diff
@@ -39,35 +31,47 @@ different applications can adapt to their own needs.
 
 While authentication is a key goal of this working group, it is not
 the objective of this working group to develop new authentication
-technologies.  Rather, the security protocol developed by this
-group will provide a way to leverage existing authentication
-technologies to associate identities with keys used in the protocol,
-just as TLS does with X.509.
+technologies.  Rather, the MLS protocol provides a way to leverage
+existing authentication technologies to associate identities with
+keys used in the protocol, just as TLS does with X.509.
 
-In developing this protocol, we will draw on lessons learned from
-several prior message-oriented security protocols, in addition to
-the proprietary messaging security protocols deployed within
+While developing the MLS protocol, the group drew on lessons learned
+from several prior message-oriented security protocols, in addition
+to the proprietary messaging security protocols deployed within
 existing applications:
 
 o S/MIME - https://tools.ietf.org/html/rfc5751
 o OpenPGP - https://tools.ietf.org/html/rfc4880
 o Off the Record - https://otr.cypherpunks.ca/Protocol-v3-4.1.1.html
 o Double Ratchet - https://en.wikipedia.org/wiki/Double_Ratchet_Algorithm
 
-The intent of this working group is to follow the pattern of
-TLS 1.3, with specification, implementation, and verification
-proceeding in parallel.  By the time we arrive at RFC, we
-hope to have several interoperable implementations as well
-as a thorough security analysis.
-
-The specifications developed by this working group will be
-based on pre-standardization implementation and deployment
-experience, and generalizing the design described in:
-
-o draft-omara-mls-architecture
-o draft-barnes-mls-protocol
+The working group followed the pattern of TLS 1.3, with specification,
+implementation, and verification proceeding in parallel.  When we arrived
+at RFC, we had several interoperable implementations as well as a thorough
+security analysis.
 
 Note that consensus is required both for changes to the protocol mechanisms
 from these documents and retention of the mechanisms from them. In particular,
 because something is in the initial document set does not imply that there is
 consensus around the feature or around how it is specified.
+
+Now that MLS has been published the group will work on the following MLS
+protocol extensions:
+
+* Support for use of MLS in protocols developed by the MIMI working group
+* Support for new credential types
+* Support for common operational patterns in messaging applications
+* Support for Post-quantum
```

Comment:
```suggestion
* Support for quantum-safe cryptographic algorithms
```

Reply:
If it is the word 'post-quantum' that the suggestion is about, then a change to "Support for quantum resistance" should suffice.